graphics/ctpv/Portfile

```
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github   1.0
+PortGroup           makefile 1.0
+PortGroup           compiler_blacklist_versions 1.0
+      
+github.setup        NikitaIvanovV ctpv 41507944c49c4bbbf8f3a21dde42243c6b4e1b79
+github.tarball_from archive
+
+license             MIT
```

According to the `port lint` output, you are missing one required stanza, `categories`.  As per the path of the Portfile, I see this should be `graphics`.  Please add the missing line to this same commit.

```suggestion
categories          graphics
license             MIT
```